README.md

```diff
@@ -15,6 +15,13 @@ conda install -c conda-forge tqdm
 conda install scipy
 ```
 
+To use `spline' flows, the third-party package *torchsearchsorted* must be installed.
+Download the source from the [github repo](https://github.com/aliutkus/torchsearchsorted) and, with the conda environment activated, run
```

Comment:
is using the torch search sorted absolutely necessary compared to using numpy version? This is still all on CPUs right? The line in their readme concerns me: "This function has not been heavily tested. Use at your own risks"

The reason I ask is I was hoping we could make a conda package and pip dependencies are not supported.

Reply:
I haven't looked very hard but it seems for us the upside of using it would be if we were on GPUs, whereas the benchmarking seems to be identical on CPU, I guess at the cost of you having to cast to some numpy type.

Reply:
I believe it was necessary because the speedup was factors of 100 compared to numpy: see my (not massively scientific) comparison [here](https://github.com/bayesiains/nflows/issues/19). However, it looks like pytorch now have their own implementation so one of the things I'm going to change now is to switch to that.

Reply:
All that said, I have to defend torchsearchsorted just for the fact that it's super easy to install and has worked like a charm so far - results are as expected and hella faster than numpy, although we never got to take advantage of the GPU implementation.

Reply:
Not disputing that, it's just that we can't build a package which depends on something needing to be installed from source. Anyway it's not super important